laineJavaPM/src/lv/laine/Person.java

```diff
@@ -0,0 +1,86 @@
+package lv.laine;
+
+import java.time.LocalDate;
+import java.time.Period;
+
+public class Person {
+    //LocalDate dob = LocalDate.of(1991, 04, 19);
+//    LocalDate date = LocalDate.now();
+//    Period diff = Period.between(dob, LocalDate.now());
```

Comment:
Comment is good when you have info about code, if you not use this code just delete